roles/st2/tasks/main.yml

```diff
@@ -57,6 +58,8 @@
   with_dict: "{{ st2_config }}"
   loop_control:
     loop_var: _conf_section
+    # prevent logging passwords, auth URIs, and secrets
+    label: "conf | {{ _conf_section.key }}"
```

Comment:
Do we still need this part, having `st2_config_no_log`?

Reply:
I think so, because the value for this option includes the entire unfiltered sub dictionary of st2_config.

Reply:
Just to understand, where it leaks? Debug mode?
Additionally, what `conf` does in this construction: `conf | {{ _conf_section.key }}`?

Reply:
~It might even be good to no_log this, as it is covered in the output of the included file~
Actually this doesn't do anything. All I get is "included: /opt/theatro/etc/ansible/roles/StackStorm.stackstorm/roles/st2/tasks/config.yml for harmony" over and over whether or not this line is here. I'll drop it.